packages/@react-native-windows/telemetry/src/test/errorUtils.test.ts

```diff
@@ -155,14 +155,8 @@ test('sanitizeErrorMessage() multiple known paths', () => {
     ),
   ).toBe(`Cannot find module react-native/package.json
```

Comment:
You can also make this value auto update is you change "toBe" to "toMatchInlineSnapshot"